src/AuthenticatedAPIClient/authInterface.js

```diff
@@ -144,16 +126,29 @@ export default function applyAuthInterface(httpClient, authConfig) {
         // Attempt to refresh the JWT cookies.
         httpClient
           .refreshAccessToken()
-          // Successfully refreshed the JWT cookies, fire the callback function.
+          // Successfully refreshed the JWT cookies
           .then((response) => {
             const refreshedAccessToken = httpClient.getDecodedAccessToken();
 
-            /* istanbul ignore next */
             if (refreshedAccessToken === null) {
-              logMissingAccessToken(httpClient, accessToken, response);
+              // TODO: Why does the api call sometimes succeed, but the JWT cookie isn't found?
+              const error = new Error('Access token is null after refresh.');
+              const cookieFound = global.document.cookie
+                .includes(httpClient.accessTokenCookieName);
+              const cookieDisabled = global.navigator &&
+                global.navigator.cookieEnabled !== undefined && !global.navigator.cookieEnabled;
+              httpClient.loggingService.logError(error, {
+                previousAccessToken: accessToken,
+                axiosResponse: response,
+                cookieDisabled,
+                cookieFound,
+              });
+              // Logs the user out to clean-up authentication and redirects to login.
+              // Pass 'refresh-token-missing' in case logout causes loss of above logError.
+              httpClient.logout(authConfig.appBaseUrl + route, 'refresh-token-missing');
```

Comment:
**Reviewer Note:** I'm not sure why this was showing 100% coverage locally.  I may have a questino about adding a test.  I am more concerned with testing this logout call.